### PR TITLE
Added support for GCP MP

### PR DESCRIPTION
**Pull request by gmflau**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -24,6 +24,22 @@ env:
 	@rm -rf .env
 	@./env.sh
 
+# help:
+.PHONY:	tf-deploy
```
Comment (tylerhutcherson) [inline Makefile:28]: @gmflau How do you feel about making this `make` command something like: `make redis-deploy`? so it's clear what's being deployed by TF.

File: README.md [hunk 2]
```diff
@@ -84,20 +81,43 @@ Make the env file and enter values as prompted. See template below:
...
+cat <<EOF >> .env
+REDISCLOUD_ACCESS_KEY=<Redis Enterprise Cloud Access Key>
+REDISCLOUD_SECRET_KEY=<Redis Enterprise Cloud Secret Key>
+REDIS_SUBSCRIPTION_NAME=<Name of your Redis Enterprise Subscription>
+REDIS_SUBSCRIPTION_CIDR=<Deployment CIDR for your Redis Enterprise Subscription cluster, ex. 192.168.88.0/24>
+EOF
+```
+3. Run the following command to deploy your Redis Enterprise database instance
+```cmd
+$ make tf-deploy
```
Comment (tylerhutcherson) [inline README.md:118]: Nice! See above note about make command name

**Review by tylerhutcherson — APPROVED** [review]
Looks great. Made a suggestion about make command naming.

File: Makefile (hunk 1 above)
Comment (gmflau) [reply Makefile:28]: Yeah.  redis-deploy and redis-destroy would work.

**Pull request merged**